configs/_base_/models/dest_simpatt-b0.py

```diff
@@ -5,20 +5,21 @@
     type='EncoderDecoder',
     pretrained=None,
     backbone=dict(
-        type='SimplifiedTransformer',
-        img_size=(1024, 1024),
-        in_chans=3,
-        num_classes=19,
+        type='SimplifiedVisionTransformer',

```

Comment:
SimplifiedMixTransformer?  since ours is based on Mix Transformer from Segformer